lib/menu.rb

```diff
@@ -0,0 +1,25 @@
+require_relative "menu_item"
+require "csv"
+
+class Menu
+
+  DEFAULT_FILENAME = "./lib/takeaway_menu.csv"
+  attr_reader :m_items
+
+  def initialize
+    @m_items = {}
+  end
+
+  def load_from_csv
+    @m_items = {}
+    CSV.foreach(DEFAULT_FILENAME, headers: true) do |row|
+      @m_items[row["number"]] = MenuItem.new(number: row["number"], dish: row["dish"], price: row["price"])
```

Comment:
Seeing a call to .new within a method is a bit of a red flag - you're creating a very hard dependency on your MenuItem class.

What this means is that the Menu class can only ever operate in conjunction with MenuItem, and therefore probably knows more than it should.  You could use dependency injection here to loosen this coupling.